client/lib/infrastructure/application/application_dto.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'application_dto.freezed.dart';
+part 'application_dto.g.dart';
+
+@freezed
+abstract class ApplicationDto implements _$ApplicationDto {
+  const ApplicationDto._();
+
+  const factory ApplicationDto({
+    required String schoolTranscript,
+    required String mainEssay,
+    required String extraEssay,
+    required List<dynamic> departmentSelection,
+    required String extraCertification,
+    required String proficencyTest,
+    required String recomendationLetter,
+    required String militaryFamilyStatus,
+    required String universityFamilyStatus,
+  }) = _ApplicationDto;
+
```

Comment:
We're gonna need a toDomain function here the takes in the DTO & returns Application